### DISCUSSION TITLE
CSS

### DISCUSSION THREAD

**Discussion by c00lkidd9** [monkeytypegame/monkeytype#2287] [Questions] [answered]
Your CSS and GUI altogether on the site look beautiful. I was wondering if I could implement something like that into a site of mine! What I mean is I need help using the themes that you guys have on my site so it could work. Any way this is possible?

**Comment by Miodec** [comment, marked as answer]
The themes work by changing the values of CSS variables. Simply use the variables when defining your classes, and add a second link tag to your html which points to the theme.

**Reply by c00lkidd9** [reply]
Thanks!
